client/dashboard/.eslintrc.js

```diff
@@ -30,8 +30,9 @@ module.exports = {
 							'@automattic/components/*',
 							'!@automattic/components/src',
 							'@automattic/components/src/*',
-							'!@automattic/components/src/summary-button',
 							'!@automattic/components/src/core-badge',
+							'!@automattic/components/src/breadcrumbs',
```

Comment:
> Clicking the breadcrumb items will trigger a page reload. This is because it's a regular link, not a router link. I don't have any good idea yet how to solve it. Need help from @Automattic/io.

How difficult is it to just create the breadcrumb links ourselves? In this case, it looks like there will only ever be a single breadcrumb link, looking at the designs, so this doesn't seem that complex to add a single `{link} /` above the header?

Reply:
Yes, if we're only ever going to add one link then we can remove that component eventually. I don't think we should necessarily import styles from Calypso components, we should focus on using `@wordpress` styles with limited customisation.

Reply:
Okay, this is not what I anticipated; I thought the idea is to make use of the reusable DS components. I'll see what I can do next week; it's getting late for me 😄

Reply:
Or I can try to pass something to the link's `onClick` props and see if I can properly call TanStack router navigation with it.

https://github.com/Automattic/wp-calypso/blob/bb42801b356f24683f2a882b5f3fafb67d0b2cc9/packages/components/src/breadcrumbs/index.tsx#L45

I'd love to get input from @ntsekouras as the component author 🙏

Reply:
Apparently Breadcrumbs was created for the hosting dashboard, but I don't have more information than that, so ignore my comments for now.

Reply:
>Or I can try to pass something to the link's onClick props and see if I can properly call TanStack router navigation with it.

For start that what I'd try to do myself, just for now.

`Breadcrumbs` will be part of the [PageHeader component](https://github.com/Automattic/wp-calypso/pull/103159), that will land soon. My plan was to implement what this PR does, after the merge of PageHeader and  try to see if we can make it work with `a` tags, but if it can't we'll add support in the Breadcrumbs component to pass the element we want instead of `a` ( in this case `Link`).

So, for me the important part here is to create the functionality to retrieve the proper breadcrumb items and for now we could just use directly a Link and not the Breadcrumbs if not possible.

Reply:
I opened a PR for this: https://github.com/Automattic/wp-calypso/pull/103321

Reply:
> see if I can properly call TanStack router navigation with it.

If you do want to shortcut #103321 you can implement the `onClick` handler like

```tsx
export default function Breadcrumbs() {
	const matches = useMatches();
	const navigate = useNavigate();

	const items = [] as BreadcrumbItemProps[];

	let hasParent = false;
	matches.forEach( ( match ) => {
		if ( match.loaderData?.breadcrumbItemLabel || hasParent ) {
			items.push( {
				label: match.loaderData?.breadcrumbItemLabel || '',
				href: match.pathname,
				onClick: ( e ) => {
					e.preventDefault();
					navigate( { to: match.pathname } );
				},
			} );
			hasParent = true;
		}
	} );

	return <BreadcrumbsComponent items={ items } />;
}
```

Reply:
@p-jackson The downside of this approach is that it doesn't support "preloading". See #103283 and same issue also being solved for DataViews #103328

Reply:
Thanks all. I understand that we're discussing how we will support route links for the breadcrumbs in the linked PR above. To keep progressing, can we then move forward with the existing Breadcrumbs component behavior in this PR, and then we can follow-up later when we have decided how to support route links? At least, in this PR, we managed to collect the breadcrumbs and show them in the appropriate location.